### PR TITLE
fix playbook name

### PR DESCRIPTION
**Pull request by erezh31**

### CODE REVIEW

**Comment by erezh31** [comment]
@aviadl do you have answer?
@asafshen maybe you know?

**Comment by asafshen** [comment]
@erezh31 aviad is right , we should not change playbook ID

**Comment by erezh31** [comment]
@meirwah plz approve & merge (CI failed on other tests)

File: Playbooks/playbook-DeDup_incidents.yml [hunk 1]
```diff
@@ -190,3 +194,4 @@ inputs:
 outputs:
 - contextPath: isSimilarIncidentFound
   description: Is similar incident found? (true\false)
+releaseNotes: "-"
```
Comment (meirwah) [inline Playbooks/playbook-DeDup_incidents.yml]: add here that you rename this playbook

**Comment by erezh31** [comment]
@meirwah can you approve?

**Comment by meirwah** [comment]
approved

**Comment by dorsha** [comment]
Build fails.

**Comment by meirwah** [comment]
@erezh31 can you update branch ?

**Comment by erezh31** [comment]
@meirwah updated

**Comment by meirwah** [comment]
plz merge when passes

**Pull request merged**